inscriptions/ISic000527.xml

```diff
@@ -190,14 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#AA">
+                <ab>
+                    <lb n="1"/>Diis M<supplied reason="lost">anibus</supplied>
```

Comment:
If you can face it, some of this is actually reported by earlier editors: compare the edition of Gualtherus (pasted image here) with Mommsen:
![image](https://user-images.githubusercontent.com/25080386/117422401-78c1af80-af17-11eb-8ef1-f087c56a0d3e.png)

The bits such as 'anibus' that Gualtherus saw, should be `<supplied reason="undefined" evidence="previousEditor">`, which will result in them being underlined in the Leiden version.